Add download attribute to Button and UnstyledLink (#1027)
Shopify/polaris-react: UNRELEASED.md

### Enhancements

> - Added `download` prop to `Button` and `UnstyledLink` components to let you set the download attribute ([#1027](https://github.com/Shopify/polaris-react/pull/1027))

Reviewer: ```suggestion
- Added `download` prop to `Button` and `UnstyledLink` components that enables the download attribute ([#1027](https://github.com/Shopify/polaris-react/pull/1027))
```

Author: "that enables" feels like it's missing a word. "... components that enables setting the download attribute" feels nicer to me

Reviewer: I was mostly just trying to avoid the use of `you`, but I don't feel strongly about it.

Author: "that enables setting" instead of "that enables" still removes the usage of "you" :)
